application/views/user/add.php

```diff
@@ -120,6 +120,15 @@
             </select>
         </div>
 
+		<div class="form-group">
+			<label for="sotalookup">SOTA auto lookup gridsquare and name for summit.</label>
+			<select class="custom-select" id="sotalookup" name="user_sota_lookup">
+				<option value="0">No</option>
+				<option value="1">Yes</option>
+			</select>
+			<div class="small form-text text-muted">If this is set, name and gridsquare is fetched from the API and filled in location and locator.</div></td>
+		</div>
+
```

Comment:
> I would have waited with this feature if I had touched the QSO view, as I know there are 2 PR's waiting for these (hopefully they'll be merged soon).

I see. But I actually meant to start adding translations with new features. I.e., to do something like 

```suggestion
		<div class="form-group">
			<label for="sotalookup"><?php echo $this->lang->line('sota_lookup_label'); ?></label>
			<select class="custom-select" id="sotalookup" name="user_sota_lookup">
				<option value="0"><?php echo $this->lang->line('general_word_no'); ?></option>
				<option value="1"><?php echo $this->lang->line('general_word_yes'); ?></option>
			</select>
			<div class="small form-text text-muted"><?php echo $this->lang->line('sota_lookup_helptext'); ?></div></td>
		</div>

```

and move those new _text lines_ to the appropriate lang file.

That way, there is not needed to _work twice_ to achieve the translations goal.

Just my two cents :)

Reply:
Thanks for enlightening me. I have not studied the translations too much. I did not think about general words and the general_words_lang file. I added translation for the dropdown now.

Perhaps a feature freeze until all translations are done would be the best. Not that I'm the one to decide this.